modules/affile/wildcard-source.c

```diff
@@ -141,7 +141,9 @@ _create_file_reader(WildcardSourceDriver *self, const gchar *full_path)
 static void
 _handle_file_created(WildcardSourceDriver *self, const DirectoryMonitorEvent *event)
 {
-  if (g_pattern_spec_match_string(self->compiled_pattern, event->name))
+  if (g_pattern_spec_match_string(self->compiled_pattern, event->name)
```

Comment:
Style check fails on this, please correct

<img width="822" height="266" alt="Screenshot 2025-07-15 at 8 43 19" src="https://github.com/user-attachments/assets/c7281b2d-f5d9-4f4a-a513-5cfc7463f2d6" />